engine/servermaster/jobop/config.go

```diff
@@ -23,9 +23,9 @@ const (
 	// If a job keeps failing, the max back interval is 5 minutes, and 10 minutes
 	// can keep at least one failed record.
 	defaultBackoffResetInterval = 2 * defaultBackoffMaxInterval
-	// backoff for 24 times, from 5s to 300s at most, with 1.2 as multiplier will
-	// cost approximately 32 minutes.
-	defaultBackoffMaxTryTime = 24
+	// with 1.2 as multiplier, it will cost approximately 32 minutes to reach max interval 5min,
+	// and it will keep trying for every 5min untill approximate 48 hours. Then, it will quit.
+	defaultBackoffMaxTryTime = 600
```

Comment:
SGTM. @amyangfei Please confirm that are there any other potential problems.

Reply:
It just leaves some unreleased resource , no other problems.